recipes/jarvis3/build.sh

```
@@ -0,0 +1,16 @@
+#! /bin/bash
+export C_INCLUDE_PATH=$C_INCLUDE_PATH:${PREFIX}/include
+export LIBRARY_PATH=$LIBRARY_PATH:${PREFIX}/lib
+cd src 
+make CC=$CC CFLAGS="$CFLAGS -fcommon"
+mkdir -p ${PREFIX}/bin
+make
+mv extra/ ${PREFIX}/bin
+mv JARVIS3.sh ${PREFIX}/bin
+mv JARVIS3 ${PREFIX}/bin
+cd ${PREFIX}/bin
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)